### DISCUSSION TITLE
Proper SystemD service

### DISCUSSION THREAD

**Discussion by MihailGershkovich** [apache/superset#14108] [Ideas]
I've created a simple SystemD service with gunicorn.
Right now I am forwarding to it from another instance, thus my settings are probably not the best. Nevertheless, probably you can add it to the documentation or to your files sooner or later:

/bin/superset.gunicorn
```
#! /bin/bash
gunicorn \
      -w 10 \
      -k gevent \
      --timeout 120 \
      -b  0.0.0.0:80 \
      --limit-request-line 0 \
      --limit-request-field_size 0 \
      --statsd-host localhost:8125 \
      "superset.app:create_app()"
```
      

/etc/systemd/system/superset.service
```
[Unit]
Description=Apache Superset
After=network-online.target
Wants=network-online.target systemd-networkd-wait-online.service

[Service]
Restart=on-failure
RestartSec=5s
ExecStart='/bin/superset.gunicorn'

[Install]
WantedBy=multi-user.target
```

feel free to ask questions, I'll try to respond in a reasonable time.

**Comment by kayhanyuksel** [comment]
Hi, we want to install gunicorn and use superset as a service, is there a guide with all tasks in order to do that, starting with the installation of pre-installation requirements of gunicorn, things to do before installation  ? 
kind regards
